### ISSUE TITLE
Use relative migration filename

### ISSUE THREAD

**Issue by jdachtera** [mycodeself/mongo-migrate-ts#122]
Hi @mycodeself,

Thank you for creating this library—it's been very helpful.

I noticed that the absolute filename of a migration is currently saved to the database. This setup means a rollback can only be performed from the same machine that executed the migration, or one with a similar directory structure.

Wouldn't it be more flexible to store the path relative to migrationsDir? This change would allow rollbacks from any machine. If you're open to PRs, I’d be happy to contribute one.

Thanks for considering this improvement.

Cheers,
@jdachtera

**Comment by mycodeself** [comment]
Hello @jdachtera,

I think you are totally right and this is a design flaw I made. The only thing that worries me a bit about this change is that it will be a breaking change, no?

Maybe we can implement a command to migrate the paths and make it easier for users, WDYT? 

Thanks a lot for your help 😃

**Comment by jdachtera** [comment]
Yeah, I think implementing that command is a great idea. I've opened a pull request which implements the changes here: #123. I'm currently looking into the tests now.

**Comment by hrithiqball** [comment]
@mycodeself can you merge the pr? or is this project no longer maintained

**Comment by mycodeself** [comment]
hi @hrithiqball this project is maintained as a best effort.

I've been quite busy, hopefully in the next few weeks I will be able to review the outstanding issues and release a new version fixing mainly third-party vulnerabilities.

**Comment by hrithiqball** [comment]
no problem! you are amazing and i enjoyed using it
